architecture/porting_guidelines.rst

```diff
@@ -58,9 +58,9 @@ variables (``CFG_``), which may be assigned values in two ways. ``CFG_FOO ?=
 bar`` should be used to provide a default value that may be modified at compile
 time. On the other hand, variables that must be set to some value and cannot be
 modified should be set by: ``$(call force,CFG_FOO,bar)``. - Compiler flags for
-the TEE core, the user mode libraries and the Trusted Applications, which may be
```

Comment:
There should be a line break before "- Compiler flags" (and before "- OP-TEE configuration" above, too).

Reply:
Let me fix that as well. Sending an update shortly, stay tuned.

Reply:
I wonder if a "hint" section would be more appropriate here?

Reply:
I.e., like this instead
![image](https://github.com/OP-TEE/optee_docs/assets/6168937/43c39f57-926d-412c-97f0-8ea79497c43f)
What do you think @jforissier ?

Reply:
Well since we mention "two things" it is a bit weird to put one on a different level than the other.